### ISSUE TITLE
preferred-alias false positive for toString

### ISSUE THREAD

**Issue by zarsky-broad** [jfmengels/eslint-plugin-lodash-fp#75]
When using `_.toString`, I get the following:

```
Use `function toString() { [native code] }` instead of `toString`
```

It looks like this is because the list of aliases is built by from `aliasToReal` [here](https://github.com/jfmengels/eslint-plugin-lodash-fp/blob/master/rules/core/lodash-util.js#L18), and it has its native `toString` prototype method.

This looks like the same thing as https://github.com/jfmengels/eslint-plugin-lodash-fp/pull/67, but there hasn't been any movement there for over a year.
